eng/targets/DiaSymReaderNative.targets

```diff
@@ -0,0 +1,21 @@
+<Project>
+
+  <PropertyGroup>
+    <PlatformTarget Condition="'$(PlatformTarget)' == ''">AnyCPU</PlatformTarget>
+  </PropertyGroup>
```

Comment:
Is this related to DSRN?

Reply:
Not sure what DSRN but added a comment + linked issue to describe the reasoning for this.